src/ui/hash.js

```diff
@@ -67,19 +70,47 @@ class Hash {
         if (mapFeedback) {
             // new map feedback site has some constraints that don't allow
             // us to use the same hash format as we do for the Map hash option.
-            hash += `#/${lng}/${lat}/${zoom}`;
+            hash += `/${lng}/${lat}/${zoom}`;
         } else {
-            hash += `#${zoom}/${lat}/${lng}`;
+            hash += `${zoom}/${lat}/${lng}`;
         }
 
         if (bearing || pitch) hash += (`/${Math.round(bearing * 10) / 10}`);
         if (pitch) hash += (`/${Math.round(pitch)}`);
-        return hash;
+
+        if (this._hashName && !mapFeedback) {
```

Comment:
Nit: can we just move this block into the `else` statement on line 74 to avoid checking `mapFeedback` a second time?

Reply:
This is hardly possible, as we need `bearing` and `pitch` to be added to hash if needed. By the way, `mapFeedback` is not well documented so I don't understand if I need to take it into account when `hashName` is set.